src/s3Accounting/components/ComparisonTab.js

```diff
@@ -0,0 +1,351 @@
+import {
+  Container,
+  Dropdown,
+  Grid,
+  Header,
+  Icon,
+  Message,
+  Placeholder,
+  Segment,
+} from 'semantic-ui-react';
+import {FaEquals, FaNotEqual, FaQuestion} from 'react-icons/fa';
+import React, {useState} from 'react';
+
+import {ALL_FILE_AUDITS} from '../queries';
+import {Chart} from 'react-charts';
+import FileUploadTable from './FileUploadTable';
+import {dateTime} from '../../common/dateUtils';
+import {useQuery} from '@apollo/client';
+
+const ComparisonTab = ({match, loading, error, data}) => {
+  const [tab, setTab] = useState('matched');
+  const [dropdown, setDropdown] = useState('count_by_data_type');
+
+  const storageAnalyses =
+    data && data.allStorageAnalyses.edges.length > 0
+      ? data.allStorageAnalyses.edges[0].node
+      : {};
+
+  const {
+    loading: fileAuditsLoading,
+    error: fileAuditsError,
+    data: fileAuditsData,
+    refetch: fileAuditsRefetch,
+  } = useQuery(ALL_FILE_AUDITS, {
+    variables: {
+      storageAnalysis: storageAnalyses.id,
+      result: tab,
+    },
+  });
+
+  const audit = storageAnalyses.stats
+    ? JSON.parse(storageAnalyses.stats).audit
+    : {};
+
+  const tabContent = {
+    differ: {
+      title: ' S3 locations that have a different file than expected',
+      description:
+        'This metric is computed by counting the file hashes in the upload manifest(s) whose url and hash match a record in the S3 inventory. These represent files that were successfully uploaded to the expected location in S3.',
+      color: '#db2828',
+    },
+    matched: {
+      title: ' files that were expected but not found in S3',
+      description:
+        'This metric is computed by counting the file hashes in the upload manifest(s) that were not found in the S3 inventory. These represent file blobs that do not exist anywhere in S3 at the time of scan.',
+      color: '#21ba45',
+    },
+    missing: {
+      title: ' files in S3 with different content than expected',
+      description:
+        'This metric is computed by counting the urls in the upload manifest(s) whose hash value is different than the hash value in the S3 inventory. These items could indicate corrupted uploads or S3 objects that have been overwritten with different content since the time of upload.',
+      color: '#db2828',
+    },
+    moved: {
+      title:
+        ' files that exist in the S3 but at a different location than expected',
+      description:
+        'This metric is computed by counting the file hashes in the upload manifest(s) whose url is different than the url in the S3 inventory. These represent file blobs that exist in S3 but at different locations than specified by a user in the upload manifest(s).',
+      color: '#fbbd08',
+    },
+  };
+
+  const dropdownText = {
+    count_by_bucket: 'Bucket',
+    count_by_data_type: 'Data Type',
+    count_by_ext: 'Extension',
+    count_by_size: 'Size',
+  };
+
+  const dropdownOptions = {
```

Comment:
I'm thinking we can eventually make the drop down completely dynamic so you don't have to hard code the options. I could update the storage analysis object so that each facet (e.g. count_by_ext) has a key and text.

Reply:
Sure, the current dropdown options are temporary way supporting the dropdown, after we are settle down with the data structures, we can add a function to dynamically generate the options.